### PR TITLE
[DM-2442] [changelog] opcua device type encoding bugfix

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
opcua device types encoding bugfix

### CODE REVIEW

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: Fixed special characters encoding in OPC-UA device types
```
Comment (MWindel) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md]:
```suggestion
title: Special character encoding in OPC-UA device types is now available
```

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-24422
+version: 10.20.88.0
+---
+When OPC-UA device type was created with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed - gateway encodes and decodes special characters as expected.
```
Comment (MWindel) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md]:
```suggestion
Previously, if you created an OPC-UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed, the gateway encodes and decodes special characters as expected.  
```

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-24422
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md]:
```suggestion
ticket: DM-2442
```

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+version: 10.20.88.0
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md:15]: The change is not merged yet, don't forget to update here if the actual version differs

**Review by aaronraab — APPROVED** [review]

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2442
+version: 10.20.88.0
+---
+Previously, if you created an OPC-UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed, the gateway encodes and decodes special characters as expected.
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md]:
```suggestion
Previously, when creating an OPC UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This issue is now fixed, the gateway encodes and decodes special characters as expected.  
```

File: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: Special character encoding in OPC-UA device types is now available
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md]:
```suggestion
title: Special character encoding in OPC UA device types is now available
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**